installer/pkg/config-generator/tls.go

```diff
@@ -253,6 +255,18 @@ func (c *ConfigGenerator) GenerateTLSConfig(clusterDir string) error {
 	if _, _, err := generateCert(clusterDir, caKey, caCert, tncKeyPath, tncCertPath, cfg, false); err != nil {
 		return fmt.Errorf("failed to generate tnc certificate: %v", err)
 	}
+
+	// Cluster API cert
+	cfg = &tls.CertCfg{
+		Subject:   pkix.Name{CommonName: "cluster-apiserver", OrganizationalUnit: []string{"bootkube"}},
+		KeyUsages: x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
+		Validity:  validityTenYears,
+		IsCA:      true,
+	}
+	if _, _, err := generateCert(clusterDir, caKey, caCert, clusterAPIServerKeyPath, clusterAPIServerCertPath, cfg, false); err != nil {
```

Comment:
I'm not familiar with the cluster API, so take this with a grain of salt, but are you sure you want `appendCA` (the last argument) `false` here?  I guess I don't understand why we'd want `appendCA` false for any CA besides the root one...